src/prompt/prepare-user-content/prepare-user-content.ts

```diff
@@ -0,0 +1,58 @@
+import {
+  CARET_MACROS,
+  SELECTION_END_MACROS,
+  SELECTION_START_MACROS,
+} from "../constants";
+import { UserContentParams } from "../user-content-params";
+import { UserPromptOptions } from "../user-prompt-options";
+
+export type UserContent = {
+  ignoredSizeBeforeContext: number;
+  ignoredSizeAfterContext: number;
```

Comment:
That two props are not used, but I decided to leave them as an interface anyway to improve tests reliability